From 2aa6b085bbde9e13d3431b5a09eb35ad49965a94 Mon Sep 17 00:00:00 2001
From: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
Date: Fri, 9 Aug 2013 18:51:32 +0300
Subject: [PORT FROM MAIN] Adding new service oemtelephony

BZ: 25540 37760

The new oemtelephony service is added to the service
manager.

Change-Id: Ie9de6bfe2a4a1beeb64e2e945921df3ab7dff3d8
Orig-Change-Id: I065e08f7ecc5b6ee74a89a0d29c82790814a15f1
Category: device enablement
Domain: Telephony-Modem-Intel
Origin: internal
Upstream-Candidate: no,proprietary
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
Signed-off-by: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
---
 cmds/servicemanager/service_manager.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/cmds/servicemanager/service_manager.c b/cmds/servicemanager/service_manager.c
index a918c19..2d3baa8 100644
--- a/cmds/servicemanager/service_manager.c
+++ b/cmds/servicemanager/service_manager.c
@@ -40,6 +40,7 @@ static struct {
     { AID_RADIO, "radio.sms" },
     { AID_RADIO, "radio.phonesubinfo" },
     { AID_RADIO, "radio.simphonebook" },
+    { AID_RADIO, "oemtelephony" },
 /* TODO: remove after phone services are updated: */
     { AID_RADIO, "phone" },
     { AID_RADIO, "sip" },
-- 
1.7.4.1

